### ISSUE TITLE
Keytar not a valid win32 application

### ISSUE THREAD

**Issue by SamBouwer** [SamBouwer/any-docker#24]

**Comment by Shampra** [comment]
This one?
![image](https://github.com/SamBouwer/any-docker/assets/16141040/5ea58857-5160-407e-84ba-3006b01dec14)

**Comment by SamBouwer** [comment]
Yep. You can solve it manually by copying the keytar file from a working installation, but even with explicitly building with a specific version of the file, it does not work. Have you been able to build and use a Windows client successfully?

**Comment by Shampra** [comment]
I clean all, retry and it's seem to be ok now. Good, but I don't understand...
And "seems" because Anytype starts (or install and start) but screen remains black under the menu bar...
`not allowed to load local resource`...

**Comment by ra3orblade** [comment]
I've managed to build Anytype under Windows from source, with downloaded middleware with update.sh, you need choco, node and other bunch of stuff but it eventually works.
